src/commands/new.ts

```diff
@@ -114,20 +113,25 @@ export default class New extends Command {
       cwd: `${site}/trellis`,
     })
 
-    this.log(`Writing vault password into ${site}/trellis/.vault_pass ...`)
+    this.log(`Writing template vault password into \`${site}/trellis/.vault_pass...\``)
     fs.writeFileSync(`${site}/trellis/.vault_pass`, trellis_template_vault_pass)
```

Comment:
I don't remember do we need to check if the file exist

Reply:
it will work without checking